alpine/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/alpine/alpine.tmpl.dockerfile.
 FROM postgres:13.2-alpine
```

Comment:
```suggestion
FROM postgres:13.3-alpine
```

Reply:
We can not change it because in the previous version we changed it and we got some error . I opened an issue to track this issue
https://github.com/citusdata/docker/issues/235

Reply:
Can you note some more details to the issue you shared ?